pages/Transfer.qml

```diff
@@ -528,82 +528,199 @@ Rectangle {
 
         GridLayout {
             visible: persistentSettings.transferShowAdvanced && appWindow.walletMode >= 2
-            columns: 6
+            columns: 2
+            Layout.fillWidth: true
+            columnSpacing: 20
+            rowSpacing: 12
+            
+            ColumnLayout {
+                Layout.fillWidth: true
+                Layout.alignment : Qt.AlignTop | Qt.AlignLeft
 
-            StandardButton {
-                id: sweepUnmixableButton
-                text: qsTr("Sweep Unmixable") + translationManager.emptyString
-                enabled : pageRoot.enabled
-                small: true
-                onClicked: {
-                    console.log("Transfer: sweepUnmixableClicked")
-                    root.sweepUnmixableClicked()
+                MoneroComponents.Label {
+                    id: offlineTxSignLabel
```

Comment:
You can remove the `id` here as you don’t access it anyway. Same for other items.

Reply:
Removed from text items, but left for button (as they were already there before).